samples/excel/90-scenarios/currency-converter.yaml

```diff
@@ -0,0 +1,178 @@
+          tblSetup[col.name] = idx;
+        });
+        if (tblSetup['CCY'] === undefined || tblSetup['Price (CCY)'] === undefined || tblSetup['Trx Date'] === undefined || tblSetup['Base'] === undefined || tblSetup['Price in Base'] === undefined) {
+          return false;
+        }
+        
+        return true;
+      }
+
+      /** Request currency exchange on specific date **/      
+      function convertValue(ccyOrig, ccyBase, start, end) {
+        // GET https://api.exchangeratesapi.io/latest?symbols=SGD,USD&base=SGD&start_at=2019-10-05&end_at=2019-10-05
+        const baseUrl = "https://api.exchangeratesapi.io/latest";
+        let query = "?base=" + ccyOrig + "&symbols=" + ccyBase + "," + ccyOrig + "&start_at=" + start + "&end_at=" + end;
+        return baseUrl + query;
+      }
+
+      /** Default helper for invoking an action and handling errors. */
+      async function tryCatch(callback) {
+        try {
+          await callback();
+        } catch (error) {
+          // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+          console.error(error);
+        }
+      }
+      /* Create sample data */
+      /* Default header */
+      /* "CCY","Price (CCY)","Trx Date","Base","Price in Base" */
+      async function setupSample() {
+        await Excel.run(async (context) => {
+          context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+          const sheet = context.workbook.worksheets.add("Sample");
+          let salesTable = sheet.tables.add("A1:F1", true);
+          salesTable.name = "SalesTable";
+          salesTable.getHeaderRowRange().values = [["Product", "CCY", "Price (CCY)", "Trx Date", "Base", "Price in Base"]];
+          salesTable.rows.add(null, [
+            ["Frames", "MYR", 5000, "2019-10-05", "SGD", null],
+            ["Chains", "CNY", 12000, "2019-10-04", "SGD",null]
+          ]);
+          sheet.activate();
+        });
+      }
+    language: typescript
+template:
+    content: |+
+        <header class="ms-welcome__header ms-bgColor-neutralLighter" style='text-align: center'>
+            <h1 class="ms-font-su">Simple Currency Converter</h1>
+        </header>
+
+        <section class="ms-font-m">
+            <p>Simple currency converter shows how to read data from a cell, call currency converter API and update the result
+                in adjecent cell.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Add sample data</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>The price conversion will be provide by conversion API. Select on empty price cell in price column, and click "Convert"</p>
+            <ol>
+              <li>Select any cell in the table.</li>
```

Comment:
If we remove the need to select the table, we don't need these steps.

Reply:
Deleted.